decidim-proposals/app/controllers/decidim/proposals/admin/valuation_assignments_controller.rb

```diff
@@ -52,6 +52,12 @@ def destroy_params
         def skip_manage_component_permission
           true
         end
+
+        def after_add_evaluator_url
+          return request.referer if request.referer.present? && request.referer.include?(:proposal_id.to_s)
+
+          request.referer
```

Comment:
```suggestion
          EngineRouter.admin_proxy(current_component).root_path
```
The default behaviour should be to return to the original url